### ISSUE TITLE
ModuleNotFoundError: No module named 'hmmlearn'

### ISSUE THREAD

**Issue by SusanL82** [LINCellularNeuroscience/VAME#111]
Hello, 

I've installed VAME with the provided VAME.yaml in Anaconda with Python 3.9. 
I then start Spyder 5.1.5 from the command line from within the VAME environment. 

However, when I try to import vame (with import vame), I run into the following error:
ModuleNotFoundError: No module named 'hmmlearn'

When I check, hmmlearn is installed just fine. 

I've also tried to install Spyder within my newly created environment via anaconda navigator instead, in case of some issue with the environment. I can only update up to v3.3.6 and this crashes in a myriad of different ways before I even try anything VAME-ish, so that doesn't seem like a great solution either.

I'm pretty sure I'm doing something very basic wrong, but I can't figure out what. 
Any helpful tips would be great. 

Susan

**Comment by pavolbauer** [comment]
Hi Susan,

Thanks for trying out VAME,
Just to make sure (because you didn't describe that step) did you activate (conda activate) the newly created VAME environment before using spyder/jupyter/prompt to run VAME?
See paragraph 3 in the workflow tutorial: https://www.nature.com/articles/s42003-022-04080-7#Sec8

Best
Pavol

**Comment by SusanL82** [comment]
Yes, sorry, I wasn't specific enough there. I think I managed to solve my issues and I'm pretty sure that there is something stupid going on with my environments. I'm still pretty confused, tbh.

This is what I did to install VAME (in the command prompt):
```
git clone https://github.com/LINCellularNeuroscience/VAME.git
cd VAME
conda env create -f VAME.yaml
conda activate VAME
spyder
```

This opened my previously installed spyder 5.2.2 (from the base environment, I think). 
and then within Spyder I tried
`import vame`
That didn't work (_ModuleNotFoundError: No module named 'vame'_). 

So I then set spyder to the VAME directory that I used for installation (this was a suggestion from a different issue, iirc), and I get the  _ModuleNotFoundError: No module named 'hmmlearn'._ I'm not sure why this is the case, as I've not had to change the directory for other packages previously. 

I've now tried to install spyder 5 in my VAME environment by doing
```
conda activate VAME
conda install spyder=5
```

This gave me a startup error for not having PyQtWebEngine installed. I installed that through the anaconda navigator environments tab within the VAME environment. 
After adding that, I can run spyder from the anaconda navigator. 
If I try to run it by typing `spyder `in the command window instead it throws the following warning: 
_fromIccProfile: failed minimal tag size sanity
C:\Users\susan\anaconda3\envs\VAME\lib\site-packages\paramiko\transport.py:219: CryptographyDeprecationWarning: Blowfish has been deprecated
  "class": algorithms.Blowfish,_

I then have to set the directory to my previously cloned VAME directory again and I can import vame! 
I will try to make my way through the provided demo code to make sure that everything actually works fine and doesn't just look fine.
